integration/integration_test.go

```diff
@@ -3938,6 +3938,7 @@ func testRotateSuccess(t *testing.T, suite *integrationTestSuite) {
 		teleport.AddUser(login, []string{login})
 	}
 
+	allTypes := types.CertAuthType("all")
```

Comment:
I did consider it but I decided it against it for a couple of reasons. 1) I didn't like mixing real types with a meta type and 2) It will be removed so I wanted to limit where it was implemented. If not for the second point I might have gone the other way but I didn't immediately have a strong feeling one way or the other.

Reply:
You're right, when I account for the `all`s in the testing it makes more sense. I moved it to a const and added it to the list of types.

Reply:
Ok, actually I moved back to the middle ground because types.CertAuthTypes was being used in tests which broke so since it will be removed in a couple of releases I don't think it makes sense to rework the tests.